src/python/WMCore/Services/Rucio/Rucio.py

```diff
@@ -663,6 +663,24 @@ def getRule(self, ruleId):
             self.logger.error("Exception getting rule id: %s. Error: %s", ruleId, str(ex))
         return res
 
+    def updateRule(self, ruleId, opts):
+        """
+        _updateRule_
```

Comment:
Please remove this line. The docstring style that we have been adopting over the past years don't have this anymore (even if this makes it inconsistent with the current module)

Reply:
done